_pages/dev/ssr/server-side-rendering-in-spartacus.md

```diff
@@ -94,16 +150,20 @@ The following steps describe how to manually add SSR support so that your Sparta
 
     ```json
     {
-      "extends": "./tsconfig.json",
+      "extends": "./tsconfig.app.json",
       "compilerOptions": {
-        "outDir": "../out-tsc/app",
-        "baseUrl": "./",
-        "module": "commonjs",
-        "types": []
+        "outDir": "./out-tsc/server",
+        "target": "es2016",
+        "types": [
+          "node"
+        ]
       },
-      "exclude": ["test.ts", "e2e/src/app.e2e-spec.ts", "**/*.spec.ts"],
+      "files": [
+        "src/main.server.ts",
+        "server.ts"
+      ],
       "angularCompilerOptions": {
-        "entryModule": "src/app/app.server.module#AppServerModule"
+        "entryModule": "./src/app/app.server.module#AppServerModule"
       }
     }
     ```
```

Comment:
As part of SSR, we're also providing initialization, required since version 9.0, so there's additional line at the top of this file:
```   
   /***************************************************************************************************
   * Load `$localize` onto the global scope - used if i18n tags appear in Angular templates.
   */
   import '@angular/localize/init';```